US County Data - pandas.ipynb

```diff
@@ -2,17 +2,20 @@
  "cells": [
```

Comment:
<p>NYC not a county, hence its FIPS code is NaN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>